yarn-project/simulator/src/avm/opcodes/accrued_substate.test.ts

```diff
@@ -365,6 +359,33 @@ describe('Accrued Substate', () => {
         expect.objectContaining({ leafIndex: leafIndex.toFr(), msgHash: msgHash.toFr(), exists: true }),
       ]);
     });
+
+    it('Should correctly show false when another L1ToL2 message exists at that leaf', async () => {
```

Comment:
Other messages cannot exists at the same leaf, the name is a bit funky to me. 
Seems Closer to checking that it does not exists at the specific index 🤷